packages/components/src/components/flipcard/flipcard.ts

```diff
@@ -0,0 +1,277 @@
+ * @summary Flipcard allows for the addition of content/information on both "sides" of the card, through means of a flip animation. Used  to add dynamism and interactivity to a page.
+ * @documentation https://solid.union-investment.com/[storybook-link]/flipcard
+ * @status stable
+ * @since 3.5.0
+ *
+ * @event sd-flip-front - Emmited when the front face of the flipcard is clicked.
+ * @event sd-flip-back - Emmited when the back face of the flipcard is clicked.
+ *
+ * @slot front - The front face of the flipcard.
+ * @slot back - The back face of the flipcard.
+ * @slot mediaFront - An optional media slot which can be as a background. Dependent from gradient variant.
+ * @slot mediaBack - An optional media slot which can be as a background. Dependent from gradient variant.
+ *
+ * @cssparts base - The component's base wrapper.
+ * @cssparts front - The container that wraps the front-side of the flipcard.
+ * @cssparts back - The container that wraps the back-side of the flipcard.
+ * @cssparts frontSlotContainer - The container that wraps the front slot.
+ * @cssparts backSlotContainer - The container that wraps the back slot.
+ * @cssparts mediaFront - The container that wraps the mediaFront slot.
+ * @cssparts mediaBack - The container that wraps the mediaBack slot.
+ * @cssparts frontSecondaryGradient - The container that wraps the secondary gradient of the front side.
+ * @cssparts backSecondaryGradient - The container that wraps the secondary gradient of the back side.
+ *
+ * @cssproperties --name - Description of the flipcard.
+ */
+@customElement('sd-flipcard')
+export default class SdFlipcard extends SolidElement {
+  @query('[part="back"]') back: HTMLElement;
+  @query('[part="front"]') front: HTMLElement;
+
+  /** Determines the variant of the front face of the flipcard. */
+  @property({ type: String, reflect: true })
+  frontVariant:
+    | 'primary'
+    | 'primary-100'
+    | 'gradient-light-top'
+    | 'gradient-light-bottom'
+    | 'gradient-dark-top'
+    | 'gradient-dark-bottom' = 'primary';
+
+  /** Determines the variant of the back face of the flipcard. */
+  @property({ type: String, reflect: true }) backVariant:
+    | 'primary'
+    | 'primary-100'
+    | 'gradient-light-top'
+    | 'gradient-light-bottom'
+    | 'gradient-dark-top'
+    | 'gradient-dark-bottom' = 'primary-100';
+
+  /** Determines the ratio for the layout of the flipcard. */
+  @property({ type: String, reflect: true }) ratio: '3:4' | '16:9' = '3:4';
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  private flipFront() {
+    this.front.classList.add('clicked--front');
+    this.back.classList.add('clicked--back');
+    this.emit('sd-flip-front');
+  }
+
+  private flipBack() {
+    this.front.classList.remove('clicked--front');
+    this.back.classList.remove('clicked--back');
+    this.emit('sd-flip-back');
+  }
+
+  private handleFrontKeydown(event: KeyboardEvent) {
+    if (event.code === 'Enter' || event.code === 'Space') {
```

Comment:
I am not sure if this is the same case on all devices but on my mac, using MS Edge browser, the space bar also triggers a scroll down. This leads to the issue the flipcard moving out of my screen and flipped... 

Not sure if this needs to be a general issue in a new ticket as it might be the case in every keyboard controlled component and I am not sure about the WCAG requirements with the space bar, but I recognized it here again.

Enter key works fine.

Reply:
I am okay with removing the flip behaviour on Space and only limiting to the "Enter" key. I thought of the spacebar as more of a secondary button anyways.

Reply:
we realized yesterday, that it seems to be a user setting on my laptop, so I wouldn't remove it directly but research which keys should be / need to be activated for accessibility.